cmd/kflex/main.go

```diff
@@ -201,6 +231,17 @@ func init() {
 	createCmd.Flags().BoolVarP(&chattyStatus, "chatty-status", "s", true, "chatty status indicator")
 	createCmd.Flags().StringArrayVarP(&hookVars, "set", "e", []string{}, "set post create hook variables, in the form name=value ")
 
+	adoptCmd.Flags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "path to kubeconfig file")
```

Comment:
The reader should be told _which_ kubeconfig. There are three involved in adoption: one for the KubeFlex hosting cluster, one to read the bootstrap Secret, and one constructed for users of the ControlPlane.